packages/crayons-icon/icon-utils/icon-registry.js

```diff
@@ -0,0 +1,12 @@
+/*global globalThis*/
+export const registerIconLibrary = (items) => {
+  globalThis.iconlibs = items;
```

Comment:
I think `globalThis` points to window object . Would advice not going with window object. 
![Screenshot 2021-12-06 at 5 31 52 PM](https://user-images.githubusercontent.com/91056073/144842714-b9a12345-327e-4c43-948d-1884b25025cd.png)